packages/nodes-base/credentials/common/aws/system-credentials-utils.ts

```diff
@@ -3,15 +3,15 @@ import { Container } from '@n8n/di';
 import { ApplicationError } from 'n8n-workflow';
 
 type Resolvers = 'environment' | 'podIdentity' | 'containerMetadata' | 'instanceMetadata';
-type RetrunData = {
+type ReturnData = {
```

Comment:
While stepping through understanding, noticed this misspelling. This type is not exported and only used below.